docs/data_driven_testing.adoc

```diff
@@ -232,6 +232,31 @@ to other data variables:
 include::{sourcedir}/datadriven/DataSpec.groovy[tag=sql-data-variable-assignment]
 ----
 
+== Accessing Other Data Variables
+
+There are only two possibilities to access one data variable from the calculation
+of another data variable.
+
+The first possibility are derived data variables like shown in the last section.
+Every data variable that is defined by a direct assignment can access all
+previously defined data variables, including the ones defined through data
+tables or data pipes:
+
+[source,groovy,indent=0]
+----
+    ...
+include::{sourcedir}/datadriven/DataSpec.groovy[tag=data-variable-assignment]
+----
+
+The second possibility is to access previous columns within data tables.
+This includes columns in previous data tables in the same `where` block:
+
```

Comment:
I would add the simple `previous-column-access` variant as well before going to the full blown `previous-column-access-multi-table`

Reply:
I guess I wanted to include it and just forgot it.
Otherwise it wouldn't make sene I actually added the example.
Fixed.